test/suites/integration/tags_test.go

```diff
@@ -103,6 +103,7 @@ var _ = Describe("Tags", func() {
 			nodeInstance := instance.NewInstance(lo.ToPtr(env.GetInstance(node.Name)))
 			Expect(nodeInstance.Tags).To(HaveKeyWithValue("Name", node.Name))
 			Expect(nodeInstance.Tags).To(HaveKey("karpenter.sh/nodeclaim"))
+			Expect(nodeInstance.Tags).To(HaveKeyWithValue("karpenter.k8s.aws/ec2nodeclass", node.Name))
```

Comment:
```suggestion
			Expect(nodeInstance.Tags).To(HaveKeyWithValue("karpenter.k8s.aws/ec2nodeclass", nodeClass.Name))
```

Does that need to be the EC2NodeClass name?

Reply:
Yes - went a bit too fast with the autocomplete.